_inc/client/components/upgrade-notice-content/index.jsx

```diff
@@ -230,6 +259,7 @@ const UpgradeNoticeContent = moduleSettingsForm(
 						<Button
 							primary={ true }
 							href="https://jetpack.com/support/site-accelerator/"
+							onClick={ this.trackLearnMoreClick }
```

Comment:
While this gets triggered, it does not include the version either.

Reply:
Done!